config/sync/views.view.messages_overview.yml

```diff
@@ -0,0 +1,813 @@
+            identifier: template
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+              licence_manager: '0'
+              anonymous: '0'
+              moderator: '0'
+              administrator: '0'
+            reduce: false
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          entity_type: message
+          entity_field: template
+          plugin_id: bundle
+        combine:
+          id: combine
+          table: views
+          field: combine
+          relationship: none
+          group_type: group
+          admin_label: ''
+          operator: allwords
+          value: ''
+          group: 1
+          exposed: true
+          expose:
+            operator_id: combine_op
+            label: 'Message contains'
+            description: ''
+            use_operator: false
+            operator: combine_op
+            identifier: combine
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+              licence_manager: '0'
+              anonymous: '0'
+              moderator: '0'
+              administrator: '0'
+            placeholder: ''
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          fields:
+            nothing: nothing
+          plugin_id: combine
+      sorts: {  }
+      title: 'Messages overview'
+      header: {  }
+      footer: {  }
+      empty: {  }
+      relationships: {  }
+      arguments: {  }
+      display_extenders: {  }
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+        - user.permissions
+      tags:
+        - 'config:core.entity_view_display.message.cc_delete.mail_body'
+        - 'config:core.entity_view_display.message.cc_delete.mail_subject'
+        - 'config:core.entity_view_display.message.cc_transition_approve_proposed.mail_body'
+        - 'config:core.entity_view_display.message.cc_transition_approve_proposed.mail_subject'
+        - 'config:core.entity_view_display.message.cc_transition_propose.mail_body'
+        - 'config:core.entity_view_display.message.cc_transition_propose.mail_subject'
```

Comment:
Yeah unfortunately this is not easy to handle since it depends on view modes and not the `message_list`. We could go with a custom cache tag but that would have to be on the message_notify side since these view modes are theirs to handle.
However, especially after the subscription epic, I don't think we will often create new message templates.

Reply:
Also, even if we are to create a new template and not export it, the tags are about the view modes and these are not really going to change ever.